src/mmw/js/src/modeling/views.js

```diff
@@ -770,6 +779,10 @@ var ResultsTabContentsView = Marionette.CollectionView.extend({
     }
 });
 
+function isEditable(scenario) {
```

Comment:
Forgot to address this before merging. I wasn't sure if it made sense to make this an instance method, considering it relies on a global variable. I could rewrite this to something like `isEditableBy(user_id)` and make it an instance method on Scenario?